samples/oauth_rails/VetsApiAppeals/config/routes.rb

```diff
@@ -0,0 +1,8 @@
+Rails.application.routes.draw do
+  root 'pages#index'
+  get '/login', to: 'pages#login'
+  post '/login', to: 'pages#login'
+  get '/hlr/new', to: 'hlr#new'
+  get '/hlr/show/:id', to: 'hlr#show'
+  post '/hlr/create', to: 'hlr#create'
```

Comment:
Just curious if there is a reason we're not using restful routes here...

Reply:
Nope! I just forgot to change it after building the initial routes as I built.